kooplex-proxy/proxy/models.py

```diff
@@ -1,31 +1,46 @@
-from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Table, MetaData
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Table
 from sqlalchemy.dialects.postgresql import DOUBLE_PRECISION, TEXT, TIMESTAMP, \
     VARCHAR, INTEGER, REAL, BOOLEAN, DATE, NUMERIC
-from database import Base
+from sqlalchemy.ext.declarative import declarative_base
 
-METADATA = MetaData()
+Base = declarative_base()
 
 
-class MViewCountrySamples(Base):
+class AbstractBase(Base):
+
+    __abstract__ = True
+    @classmethod
+    def get_schema(cls):
+        return cls.__table_args__['schema']
+
+    @classmethod
+    def set_schema(cls, schema_name):
+        cls.__table_args__['schema'] = schema_name
+
+
+class MViewCountrySamples(AbstractBase):
     __tablename__ = "app_country_samples_full"
+    __table_args__ = {'schema': 'sandbox_public'}
     country = Column(VARCHAR, primary_key=True)
     n_sample = Column(INTEGER)
     log_n_sample = Column(DOUBLE_PRECISION)
     relative_n_sample = Column(DOUBLE_PRECISION)
     relative_log_n_sample = Column(DOUBLE_PRECISION)
 
 
-class MViewHumanMetaMv(Base):
+class MViewHumanMetaMv(AbstractBase):
     __tablename__ = "app_human_meta_mv"
+    __table_args__ = {'schema': 'sandbox_public'}
```

Comment:
Again, it is a bad idea to populate hardcoded schema name everywhere. You should keep the default schema name in one place, perhaps, in the parent class.